python/master.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python3
```

Comment:
Great start. This file seems to have a lot of utility functions for your histone analysis, so I would suggest renaming the file "histone_utility.py" or something along those lines. A good file name should describe its contents well.